DataFormats/GEMRecHit/src/ME0Segment.cc

```diff
@@ -38,17 +38,19 @@ ME0Segment::ME0Segment(const std::vector<const ME0RecHit*>& proto_segment, const
   theLocalDirection(direction), theCovMatrix(errors), theChi2(chi2){
   theTimeValue = 0.0;
   theTimeUncrt = 0.0;
+  theDeltaPhi  = 0.0;
   for(unsigned int i=0; i<proto_segment.size(); ++i)
```

Comment:
also, if you are at it, you can use initialization of a base or a non-static member by constructor initializer list